src/pages/home/Home.tsx

```diff
@@ -1,88 +1,127 @@
+        <IonItem>
+          <IonGrid>
+            <IonRow>
+              <IonCol size="2">
+                <IonIcon size="large" icon={document} />
               </IonCol>
-            ))}
-          </IonRow>
-        </IonGrid>
-        <IonListHeader>
-          Popular Topics
-        </IonListHeader>
-        <IonGrid>
-          <IonRow>
-            {moemorizedTopics.filter((o) => o.featured === true).map((topic, index) => (
-              <IonCol size="12" size-sm="6" size-xl="4" key={index}>
-                <IonCard routerLink={"guides/" + topic.id}>
-                  <IonCardHeader>
-                    <IonCardTitle>{topic.title}</IonCardTitle>
-                  </IonCardHeader>
-                  <IonCardContent>
-                    <IonGrid>
-                      <IonRow>
-                        <IonCol size="3">
-                          <img src={"../../assets/images/" + topic.id + ".svg"} onError={(e) => { e.currentTarget.src = "../../assets/shapes.svg" }} alt={topic.title} />
-                        </IonCol>
-                        <IonCol size="9">
-                          {parse(topic.snippet)}
-                        </IonCol>
-                      </IonRow>
-                    </IonGrid>
-                  </IonCardContent>
-                </IonCard>
+              <IonCol size="7">
+                <h4>The Constitution</h4>
+              </IonCol>
+              <IonCol size="3">
+                <IonRouterLink class="ion-float-right" href="/search">Search</IonRouterLink>
               </IonCol>
+            </IonRow>
+            <p>Browse the Constitution for the provisions that you need.</p>
+            <IonButton href="/constitution" expand="block">Browse Sections</IonButton>
+          </IonGrid>
+        </IonItem>
+        <IonItem>
+          <IonGrid>
+            <IonRow>
+              <IonCol size="2">
+                <IonIcon size="large" icon={people} />
+              </IonCol>
+              <IonCol size="7">
+                <h4>Guides</h4>
+              </IonCol>
+              <IonCol size="3">
+                <IonRouterLink class="ion-float-right" href="/search">Search</IonRouterLink>
+              </IonCol>
+            </IonRow>
+            <p>Better understand the provisions of the Constitution and their implications for parlimentary matters.</p>
+            <IonSegment value="all">
+              <IonSegmentButton value="all">
+                <IonLabel>Most Popular</IonLabel>
+              </IonSegmentButton>
+              <IonSegmentButton value="favorites">
+                <IonLabel>Recently Viewed</IonLabel>
+              </IonSegmentButton>
+            </IonSegment>
+          </IonGrid>
+        </IonItem>
+        <IonItem>
+          <IonList>
+            {guides.slice(0,3).map((guide) => (
+              <IonItem key={guide.id} routerLink={"/guides/" + guide.id}>
```

Comment:
Please replace this with the custom `TopicItem` tag, so that you don't need to worry about correct URLs, images etc.